src/main/java/controller/EditUserServlet.java

```diff
@@ -40,26 +39,20 @@ protected void doGet(HttpServletRequest request,
     @Override
     protected void doPost(HttpServletRequest request,
                           HttpServletResponse response) throws ServletException, IOException {
-        final HttpSession session = request.getSession();
-        final String roleCurrentUser = (String) session.getAttribute("roleCurrentUser");
-        if (roleCurrentUser.equals("admin")) {
-            try {
-                String email = request.getParameter("email");
-                String password = request.getParameter("password");
-                Optional<User> optUser = userService.getByEmail("email");
-                if (optUser.isPresent()) {
-                    User user = optUser.get();
-                    user.setEmail(email);
-                    user.setPassword(password);
-                    request.getRequestDispatcher("/users.jsp").forward(request, response);
-                }
-            } catch (NumberFormatException ex) {
-                request.setAttribute("validValues", "Something is wrong. Try again.");
-                request.getRequestDispatcher("/edit_user.jsp").forward(request, response);
+        try {
+            String email = request.getParameter("email");
+            String password = request.getParameter("password");
+            Optional<User> optUser = userService.getByEmail("email");
+            if (optUser.isPresent()) {
+                User user = optUser.get();
```

Comment:
try to create method update and change user here)
the same with products

Reply:
yes. i did it already with sql

Reply:
i have a new branch with sql
and now i can't modify this code
i show it you in next HW